ports/nrf/modules/machine/modmachine.c

```diff
@@ -189,6 +189,16 @@ STATIC mp_obj_t machine_disable_irq(void) {
 }
 MP_DEFINE_CONST_FUN_OBJ_0(machine_disable_irq_obj, machine_disable_irq);
 
+STATIC mp_obj_t machine_unique_id(void) {
+    uint8_t chipid[8];
```

Comment:
This does not generally guarantee 32-bit alignment as far as I know, yet you are accessing it as uint32_t. IMHO it would be simpler and cleaner to define as uint32_t...

Reply:
Thanks for the suggestion, implemented.

Regarding documentation, machine.unique_id() is standard micropython and was just an omission in the nrf port. I'm not aware of documentation that states it's not present in nrf. So I doubt there is anything to update.